src/SonarQube.Client.Tests/Api/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -96,7 +96,6 @@ public class SonarQubeService_GetSuppressedIssuesAsync : SonarQubeService_TestBa
         ""endOffset"": 107
     },
     ""flows"": [],
-    ""resolution"": """",
```

Comment:
I called the API on SonarCloud and the data returned didn't have a _resolution_ field if the status was _Open_
i.e. looks like this test JSON was wrong.

Reply:
What about a SonarQube instance like Next?